Core/Object Arts/Dolphin/Base/SequenceableCollection.cls

```diff
@@ -724,6 +724,13 @@ indexOf: target ifAbsent: exceptionHandler
 		ifTrue: [exceptionHandler value]
 		ifFalse: [index]!
 
+indexOf: target startingAt: start
+	"Answer the <integer> index of the first element of the receiver which is 
+	equal to the <Object> argument, target, within the receiver. If the receiver 
+	does not contain any elements equal to target, answer 0."
+
+	^self nextIndexOf: target from: start to: self size!
```

Comment:
Generally speaking tests should accompany base system additions, but given this is a convenience wrapper on top of `nextIndexOf:from:to:`, we can live without here.